boilerplate/app/theme/colors.ts

```diff
@@ -0,0 +1,75 @@
+// MAVERICKTODO: write documentation for colors and palette in own markdown file and add links from here
+
```

Comment:
Consolidates palette and colors into single file.

Reply:
i link it when its consolidated

Reply:
Yeah, this is nice!